website/tsconfig.json

```diff
@@ -29,7 +29,8 @@
 		"**/*.ts",
 		"**/*.tsx",
 		".next/types/**/*.ts",
-		"postcss.config.cjs"
+		"postcss.config.cjs",
+		"content/carbon/plugins/index.mdx"
```

Comment:
extra tsconfig entry?

Reply:
ah another instance of the editor updating imports and such when i dont want it to

Reply:
figured as much 😂